gen/enum.go

```diff
@@ -134,7 +141,11 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 				<end ->
 				}
 			<end ->
-			return []byte(<$strconv>.FormatInt(int64(<$v>), 10)), nil
+			<if checkEnumTextMarshalStrict ->
+				return nil, <$fmt>.Errorf("unknown value for enum %s", <$v>)
```

Comment:
we should include the value and the enum name like above:

```suggestion
				return nil, <$fmt>.Errorf("unknown enum value %q for %q", <$v>, "<$enumName>")
```